test/plugins/pipelines.test.js

```diff
@@ -856,7 +856,7 @@ describe('pipeline plugin test', () => {
     describe('POST /pipelines', () => {
         let options;
         const unformattedCheckoutUrl = 'git@github.com:screwdriver-cd/data-MODEL.git';
-        const checkoutUrl = 'git@github.com:screwdriver-cd/data-model.git';
+        const checkoutUrl = 'git@github.com:screwdriver-cd/data-model.git#master';
```

Comment:
`checkoutUrl` was only used where the scmUri should have been, so I renamed `formattedCheckoutUrl` to `checkoutUrl` for line length. Probably less clear though, reverting back to `formattedCheckoutUrl`.